core/domain/exp_domain.py

```diff
@@ -2836,6 +2858,29 @@ def _convert_v33_dict_to_v34_dict(cls, exploration_dict):
 
         return exploration_dict
 
+    @classmethod
+    def _convert_v34_dict_to_v35_dict(cls, exploration_dict):
```

Comment:
Same here, I think this needs to be tested?

Reply:
Ditto.